style.css

```diff
@@ -42,14 +41,16 @@ list-style-type: none;
 
 
 .footer {
-  background: rgb(242, 242, 242); 
-  flex-shrink: 0; 
+  background: rgb(242, 242, 242);  
   display: flex;
+  position: absolute;
+  bottom: 0;
+  width: 100%; 
+  height: 2,5rem; 
 }
 
 .footer-nav-left { 
   display: inline-flex;
```

Comment:
Bare `display: flex`. "inline"-delen siger noget om hvordan elementet indgår i den øvrige omkringliggnede kontekst og _ikke_ noget om hvad flex-delen går ud på. "Inline" betyder i en nøddeskal, at et element opfører sig som tekst gør. Husk at HTML er en gammel måde at lave tekst-dokumenter på.

Læs her om block vs inline-block. Det samme er gældende for inline-flex: https://alligator.io/css/display-inline-vs-inline-block/